mattermost-plugin/server/boards/mutex_adapter.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package boards
+
+import (
+	"errors"
+	"net/http"
+
+	mm_model "github.com/mattermost/mattermost-server/v6/model"
```

Comment:
Maybe not for this PR, but we're referring to the Mattermost Model as both `mm_model` and `mmModel`. We should pick one and standardize

Reply:
Code debt ticket created:  https://community.mattermost.com/boards/workspace/qgsck6cts3fwpqwyjiupjm5cde/b4ctmm917q3rzdfi451qtuxjzcw/v4mi5d7u6afbhfrks5783gbmj9o/csz137h5mjtgj3b68xtu5cs5zqr